.idea/misc.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Those are usually not added, garbage like the rest of the .idea dir
part of the gitignore